test/controllers/users/registrations_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Users
+  class RegistrationsControllerTest < ActionDispatch::IntegrationTest
+    test "should register new user" do
+      user = {
+        name: "name",
+        surname: "surname",
+        email: "test@example.com",
+        password: "password",
+        password_confirmation: "password"
+      }
+
+      post "/user", params: {
+        user: user
+      }
+
+      user_list = User.find_by!(name: user[:name], surname: user[:surname], email: user[:email])
```

Comment:
`find_by` returns a single user, so it is not a list. Maybe it is better to name it `user`? Or `new_user`...

Is email not enough to find a user?

Using `!`-version of this method does not make sense if you have an assert later